tests/functional/cephx_keyrotation/test_cephx_keyrotation_bootstrap_cleanup.py

```
@@ -0,0 +1,120 @@
+                for entity in constants.CEPHX_BOOTSTRAP_NON_OSD_KEYS
+                if entity in pre_bootstrap_entities
+            ]
+        )
+
+        osd_pods_before = get_osd_pods(namespace=namespace)
+        osd_count_before = len(osd_pods_before)
+        log.info(f"OSD pod count before add_capacity: {osd_count_before}")
+
+        try:
+            osd_size = get_osd_size()
+            log.info(f"Adding OSD capacity (device set size={osd_size})")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)